Sources/FluentUI_iOS/Components/Bottom Sheet/BottomSheetController.swift

```diff
@@ -677,6 +677,20 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
         }
     }
 
+    private func updateResizingHandleViewAccessibilityValue(for state: BottomSheetExpansionState) {
+        switch state {
+        case .collapsed:
+            resizingHandleView.accessibilityValue = "Accessibility.Drawer.ResizingHandle.Value.Collapsed".localized
+            break
+        case .expanded:
+            resizingHandleView.accessibilityValue = "Accessibility.Drawer.ResizingHandle.Value.Expanded".localized
+            break
+        default:
```

Comment:
General best practice to avoid default cases (especially for enums we define in our module) so any added cases must explicitly consider whether there should be an accessibility value